pages/aplicar/gracias/[bootcamp].tsx

```diff
@@ -11,113 +13,111 @@ interface Props {
   bootcamp: 'javascript-live' | 'python-live' | 'android' | 'ios' | 'backbase'
 }
 
-export default function Thankyou (props: Props) {
+export default function Thankyou(props: Props) {
   const isBackbase = props.bootcamp?.toLowerCase() === 'backbase'
     || props.bootcamp?.toLowerCase() === 'android'
     || props.bootcamp?.toLowerCase() === 'ios'
 
   return (
     <>
       <NavBar />
-      <section className='hero has-background-black-bis is-fullheight' id='thanks-page'>
-        <main className='hero-body'>
-          <div className='container has-text-centered'>
-            <div className='columns is-centered'>
-              <div className='column is-half'>
-                <div className='card has-background-grey-darker is-rounded p-5'>
-                  <div className='columns is-multiline container'>
-                    <div className='column is-full mt-5'>
-                      <H3>
-                        <span className='h3-cyan'>Gracias</span> por
-                        <br />
-                        <Cursor children='registrarte' white />
-                      </H3>
-                    </div>
-                    <div className='column is-full mt-5 mb-5'>
-                      Te contactaremos lo más
-                      <br />
-                      pronto posible
-                    </div>
-                    {isBackbase && (
-                      <>
-                        {/* Android */}
-                        <div className='column is-full mb-5 is-hidden-touch'>
-                          <Button
-                            isPrimary
-                            label='Descarga el Brochure de Android'
-                            icon='/icons/download.svg'
-                            href={'/brochures/android/desktop'}
-                          />
-                        </div>
-                        <div className='column is-full mb-5 is-hidden-desktop'>
-                          <Button
-                            isPrimary
-                            label='Descarga el Brochure de Android'
-                            icon='/icons/download.svg'
-                            href={'/brochures/android/mobile'}
-                          />
-                        </div>
+      <PageSection
+        sectionClassName='bg-brand-black h-screen'
+        contentClassName=' my-auto'
+      >
+        <div className={classNames(
```

Comment:
- Aquí podrías utilizar el componente `Card` y no ponerle `isBordered`, creo que sólo le pasarías el background y el border-radius `wide` (30px)

Reply:
- Nao, me quedé con la duda, ¿Para estas cards (thank you) nos funciona usar el componente `Card` o no? 🤔